pkg/pillar/vendor/github.com/lf-edge/eve/api/go/config/cipherinfo.pb.go

```diff
@@ -0,0 +1,164 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// source: cipherinfo.proto
```

Comment:
This .proto file doesn't exist. Maybe that is why yetus is complaining about cipherinfo?

Reply:
I have tried this also, still hitting the yetus issue. Looks like, linter false positive.